ios/RNMagicScript/components/UiNodes/UiListView/UiListViewItemNode.swift

```diff
@@ -89,8 +89,10 @@ import SceneKit
         backgroundNode.position = SCNVector3(0.0, 0.0, -0.005)
     }
 
-    @objc override func hitTest(ray: Ray) -> BaseNode? {
+    override func hitTest(ray: Ray) -> HitTestResult?  {
+        guard let child = childNode else { return nil }
         guard let _ = selfHitTest(ray: ray) else { return nil }
-        return childNode
+        guard let hitPoint = getHitTestPoint(ray: ray) else { return nil }
+        return (node: child, point: hitPoint)
```

Comment:
@pleszkiewicz if hitTest is inside `UiListViewItemNode` shouldn't we return `child.hitTest(ray: ray)`?